BUILD.md

```diff
@@ -1,6 +1,6 @@
 # Build the Wasm Module
 
-To build the wasm module, make sure you have docker version 24.0.7 or later installed and running.
+To build the wasm module, make sure you have docker version 26.1.1 or later installed and running.
```

Comment:
Why bump if we already say "or later"?

Reply:
I haven't tested it with 24.0.7, so it's just to be extra cautious. We don't need to bump it every time, I just did it because I was editing the file anyway